### ISSUE TITLE
edupub heading with alt-less image not a violation

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epubcheck#1110]
There's also a test that reports a heading as invalid if it only contains an img tag with no alt attribute.

While I conceptually agree this is unusable, and violates WCAG, it's not a requirement of the edupub specification that headings have text content so I can't match it up to a specific section.
